common/scripts/buildAll.sh

```diff
@@ -3,14 +3,9 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Comment:
I believe that don't need this script anymore, as the user can just run `./gradlew publishToMavenLocal` in the root directory.

Reply:
I reintroduced it _if_ we want users to be able to execute the samples themselves. We may want to provide something different, though.

Reply:
Well, users could:
```
gradle publishToMavenLocal
cd samples/<sampleName>
gradle nativeBuild
```
right?